xunit-autolabeler-v2/ast_parser/python/__init__.py

```diff
@@ -11,3 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
+# Add the parent 'ast_parser' module to PYTHONPATH
+# for consistency between Python and pytest
+#  Pytest root dir: '$REPO_ROOT/'
+#  Python root dir: '$REPO_ROOT/ast_parser'
+import sys
+sys.path.append('..')
```

Comment:
I'm still not a fan of this hack, but I can live with this considering it has a comment for the purpose of this hack.